.github/docker/centreon-web/alma9/entrypoint/container.d/80-register_pollers_background.sh

```
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+while true ; do
+  sleep 10
+
+  SQL_RESULT=$(timeout 5 mysql -h${MYSQL_HOST} -uroot -p"${MYSQL_ROOT_PASSWORD}" centreon -e "SELECT id FROM nagios_server WHERE name NOT IN (SELECT name from centreon_storage.instances)")
+  if [[ "$SQL_RESULT" == *"id"* ]] ; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)